config/blocks/footer.yml

```diff
@@ -0,0 +1,47 @@
+corporate_footer:
```

Comment:
So we discussed with @sergepavle and we could turn this into a service parameter, to be injected into the block, so that we can override it in the theme and test the block with a fixture, instead of the data contained by the module. This will allow us to update links freely in the module without breaking tests on the theme.

Reply:
Done